src/main/java/seedu/address/storage/StorageManager.java

```diff
@@ -7,23 +7,23 @@
 
 import seedu.address.commons.core.LogsCenter;
 import seedu.address.commons.exceptions.DataConversionException;
-import seedu.address.model.ReadOnlyAddressBook;
+import seedu.address.model.ReadOnlyData;
 import seedu.address.model.ReadOnlyUserPrefs;
 import seedu.address.model.UserPrefs;
 
 /**
- * Manages storage of AddressBook data in local storage.
+ * Manages storage of Data data in local storage.
```

Comment:
uh this is really weird - i think we can do better than calling it Data.

Reply:
im calling it data cause its going to be a generic for Persons, competitions, and particpation

Reply:
i just havent gotten to it yet

Reply:
It would be better to implement that generic first before merging this PR then.

Reply:
not really. currently its just a name change. i havent actually worked on the generic yet

Reply:
I see. I think it would be better to leave the name as is first, rather than rename it when it doesn't reflect the intended purpose yet.